cmd/cluster.go

```diff
@@ -542,6 +557,25 @@ func clusterCreate() error {
 		log.Fatalf("provided cri:%s is not supported, only docker or containerd at the moment", cri)
 	}
 
+	var (
+		clusterAudit  bool
+		auditToSplunk bool
+	)
+	switch audit {
+	case "off":
+		clusterAudit = false
+		auditToSplunk = false
+	case "on":
+		clusterAudit = true
+		auditToSplunk = false
+	case "splunk":
+		clusterAudit = true
+		auditToSplunk = true
+	case "":
```

Comment:
So when empty string is passed, this equals off? Why not just allow on/off/splunk?